src/main/scala/no/gdl/bookapi/controller/BooksController.scala

```diff
@@ -145,6 +164,29 @@ trait BooksController {
         Sort.valueOf(paramOrNone("sort")).getOrElse(Sort.ByIdAsc))
     }
 
+    get("/mine/?", operation(getMyBooks)) {
+      requireUser
+
+      val pageSize = intOrDefault("page-size", 10).min(100).max(1)
+      val page = intOrDefault("page", 1).max(1)
+      val sort = Sort.valueOf(paramOrNone("sort")).getOrElse(Sort.ByIdAsc)
+      val lang = LanguageTag(DefaultLanguage)
+
+      readService.withLanguageAndLevel(lang, None, pageSize, page, sort)
+    }
+
+    get("/mine/:id/?", operation(getMyBook)) {
+      requireUser
+
+      val id = long("id")
+      val lang = LanguageTag(DefaultLanguage)
+
+      readService.withIdAndLanguage(id, lang) match {
+        case Some(x) => x
+        case None => NotFound(body = Error(Error.NOT_FOUND, s"No book with id $id and language $lang found"))
```

Comment:
Does it make sense to report about language here, as the user never specified it?